visualization/package.json

```diff
@@ -1,174 +1,174 @@
+		"package.json",
+		"package-lock.json",
+		"README.md",
+		"tsconfig.json",
+		"cli.js"
+	],
+	"codecharta": {
+		"apiVersion": "1.2"
+	},
+	"window": {
+		"icon": "app/icon.png",
+		"toolbar": true,
+		"frame": true,
+		"width": 1024,
+		"height": 768
+	},
+	"main": "dist/webpack/index.html",
+	"build": {
+		"nwVersion": "0.50.3",
+		"output": "./dist/packages/",
+		"outputPattern": "codecharta-visualization-${VERSION}-${PLATFORM}-${ARCH}",
+		"files": "./dist/webpack/**/*",
+		"packed": true,
+		"targets": [
+			"zip"
+		],
+		"appId": "io.github.maibornwolff.codecharta",
+		"win": {
+			"productName": "codecharta-visualization",
+			"companyName": "MaibornWolff GmbH",
+			"icon": "app/assets/icon.ico"
+		},
+		"mac": {
+			"icon": "app/assets/icon.icns",
+			"displayName": "codecharta-visualization"
+		}
+	},
+	"browserslist": [
+		"last 2 versions",
+		"not IE 11"
+	],
+	"scripts": {
+		"start": "nw .",
+		"build": "webpack --config ./conf/webpack.config.js --env STANDALONE=true",
+		"build:web": "webpack --config ./conf/webpack.config.js --env STANDALONE=false",
+		"dev": "webpack serve --config ./conf/webpack.config.js --env STANDALONE=false",
+		"test": "jest --config ./jest.config.json .*\\.spec\\.ts --env=jsdom",
+		"test:auto": "npm test -- --watch",
+		"test:autoNoCoverage": "npm run test:auto -- --coverage false",
+		"test:updateSnaps": "npm test -- --updateSnapshot",
+		"e2e": "jest --config ./jest.config.json .*\\.e2e\\.ts --coverage false",
+		"e2e:auto": "npm run e2e -- --watch",
+		"e2e:ci": "npm run e2e -- --runInBand",
+		"package": "rimraf dist/packages && build --concurrent --tasks win-x86,win-x64,linux-x86,linux-x64,mac-x64 --mirror https://dl.nwjs.io/ . && bestzip ./dist/packages/codecharta-visualization-$npm_package_version-web.zip ./dist/webpack",
+		"plop": "tsc --esModuleInterop true --outDir ./dist/plop ./plopfile.ts && cd dist/plop && plop",
+		"lint": "eslint \"app/**/*.ts\"",
+		"lint:fix": "npm run lint -- --fix",
+		"precommit": "lint-staged",
+		"schema:generate": "typescript-json-schema ./app/codeCharta/codeCharta.model.ts ExportCCFile --required --topRef --out app/codeCharta/util/generatedSchema.json"
+	},
+	"lint-staged": {
+		"app/**/*.ts": "eslint --fix"
+	},
+	"bin": {
+		"codecharta-visualization": "cli.js"
+	},
+	"license": "BSD-3-Clause",
+	"dependencies": {
+		"ajv": "^6.12.2",
+		"angular": "^1.8.0",
+		"angular-animate": "^1.6.9",
+		"angular-aria": "^1.6.9",
+		"angular-material": "^1.1.7",
+		"angular-mocks": "^1.6.9",
+		"angular-sanitize": "^1.7.9",
+		"angularjs-nouislider": "^14.1.1",
+		"angularjs-slider": "^7.0.0",
+		"color-convert": "^2.0.0",
+		"core-js": "^3.6.5",
+		"d3-hierarchy": "^2.0.0",
+		"font-awesome": "^4.7.0",
+		"ignore": "^5.0.2",
+		"jquery": "^3.3.1",
+		"klona": "^2.0.4",
+		"lodash.camelcase": "^4.3.0",
+		"lodash.debounce": "^4.0.8",
+		"lodash.isequal": "^4.5.0",
+		"lodash.throttle": "^4.1.1",
+		"md5": "^2.3.0",
+		"nouislider": "^14.6.0",
+		"nw": "^0.50.3",
```

Comment:
Is there no change in our package-lock necessary?

Reply:
no, it seems like it was never changed to the new version to begin with
package-lock is still at 0.50.3, I think the builder update was kind of erroneous